GameFramework/GameFramework/DirectXParam/DirectXParamFactory/DirectXParamFactory.h

```diff
@@ -22,7 +22,7 @@ namespace gameframework
 		template<typename T>
 		static void Create(T** ppT)
 		{
-			pT = new T();
+			ppT = new T();
```

Comment:
`ppT` は仮引数ですね。これに直接何かを代入しても意味無いと思うのですが…
```suggestion
			*pT = new T();
```
がしたいのかな？

Reply:
*ppT = new T();
このような感じでポインタにポインタを代入するようにするつもりでした修正します

Reply:
修正しました